cluster-autoscaler/core/podlistprocessor/currently_drained_nodes.go

```diff
@@ -51,6 +51,11 @@ func currentlyDrainedPods(context *context.AutoscalingContext) []*apiv1.Pod {
 			continue
 		}
 		for _, podInfo := range nodeInfo.Pods {
+			// Filter out pods that has deletion timestamp set
+			if podInfo.Pod.DeletionTimestamp != nil {
+				klog.Infof("Pod %v has deletion timestamp set, skipping", podInfo.Pod.Name)
```

Comment:
nit: I'd slightly rephrase the log line to make it a bit clearer to anyone not  familiar with this specific part of the code. From the logs, it won't be clear what exactly is skipped for the pod. `skipping injection into the pending pod list`?

Reply:
done